README.md

```diff
@@ -1 +1,140 @@
-# Wall-Painting-Bot
+ABSTRACT
+========
+Wall painting bot is a fully automatic robot capable of sketching images on the wall upto dimensions 90cm x 120cm (approx). 
+The bot will take image through a laptop and will convert the image into 5 colors. 
+The bot will automatically change the color according to the image and will sketch on the wall.
+
+ Acknowledgements
+ ===============
+
+We would like to express our special thanks to Alok Gupta sir , Prashant Sir , Abhimanyu Sir , Nikhil sir for helping us throughout the journey.
+
+Mechanical Components-
+======================
+-Nut-bolts

```

Comment:
Write this ass numbered list like:
1. Nut-Bolts
2. Washers
3. Steel rod and so on...